src/FirstPayment/Actions/AddGenericOrderItem.php

```diff
@@ -13,16 +13,16 @@ class AddGenericOrderItem extends BaseAction
      * @param \Illuminate\Database\Eloquent\Model $owner
      * @param \Money\Money $subtotal
      * @param string $description
-     * @param int $roundedType
+     * @param int $roundedMode
```

Comment:
Can you make it `roundingMode` so it's consistent with MoneyPHP?

Reply:
Renamed